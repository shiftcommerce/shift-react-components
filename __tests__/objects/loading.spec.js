// Libraries
import React from 'react'

// Objects
import Loading from '../../src/objects/loading'

test('loading spinner renders correctly', () => {
  // Act
  const wrapper = shallow(<Loading />)

  // Assert
  expect(wrapper).toMatchSnapshot()
  expect(wrapper).toHaveClassName('o-loading')
})
