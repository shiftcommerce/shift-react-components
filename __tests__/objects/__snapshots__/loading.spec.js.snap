// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading spinner renders correctly 1`] = `
<Loading
  className="o-loading"
  color="#f5c6cb"
  delay={250}
  height={50}
  type="spin"
  width={50}
/>
`;
